src/options_parser.cc

```diff
@@ -35,11 +39,30 @@ OptionsResult* protagonist::ParseOptionsObject(Handle<Object> optionsObject) {
             else
                 optionsResult->options &= snowcrash::ExportSourcemapOption;
         }
+        else if (TypeOptionKey == *String::Utf8Value(key)) {
+            // TypeOption
+            char *strValue = *String::Utf8Value(value);
+            if (TypeOptionAst == strValue) {
+                // TODO
+                // optionsResult->options &= snowcrash::TypeOptionAst;
```

Comment:
These options will not be present in snowcrash but in drafter. I think they are already implemented in drafter.